src/App/Pages/Vault/AddEditPage.xaml

```diff
@@ -558,7 +558,14 @@
                 <StackLayout StyleClass="box-row, box-row-switch">
                     <Label
                         Text="{u:I18n PasswordPrompt}"
-                        StyleClass="box-label-regular"
+                        StyleClass="box-label-regular" />
+                    <controls:FaButton 
+                        StyleClass="box-row-button, box-row-button-platform"
+                        Text=""
```

Comment:
Please use the unicode value here instead of the actual char (for example: `&#xf24d`)

Reply:
Didn't know that was possible, but definitely preferable over the unicode character.

Reply:
Is **[this](https://fontawesome.com/v4.7/icon/question-circle-o)** the correct char?  If so we should use `&#xf29c`

Reply:
Changed to hex, not sure where I got the decimal encoded unciode character from.